test/Bruno/Altinn.Authorization/environments/AT22.bru

```diff
@@ -0,0 +1,8 @@
+vars {
+  baseUrl: https://platform.at22.altinn.cloud
+  tokenBaseUrl: {{process.env.TOKEN_BASE_URL}}
```

Comment:
tokenBaseUrl can be removed from all environments as it's not being used

Reply:
fixed